### PR TITLE
Notification box closes when clicked outside it.

### LINKED ISSUE #132: Make Notification Box Close when clicked outside it
screen shot 2018-06-20 at 1 35 54 pm

Currently only way to close it is clicked the bell icon again.

### PR DESCRIPTION
**Pull request by Rishabh570**
Notification box now closes either by clicking on notification-icon or outside notification box, anywhere on page...

Fixes #132 .
@abhishek97 Please have a look...

### CODE REVIEW

File: app/pods/components/notification-dropdown/component.js [hunk 1]
```diff
@@ -24,13 +25,22 @@ export default class NotificationDropdownComponent extends Component {
...
     super (...arguments)
+    const _this = this;
 
     this.get('loadNotifications').perform ()
 
     setInterval (() =>
       this.get ('loadNotifications').perform (),
       this.get('refreshInterval')
     )
+    document.addEventListener("click", function() {
```
Comment (abhishek97) [inline app/pods/components/notification-dropdown/component.js]: Move this code inside [`didInsertElement`](https://guides.emberjs.com/release/components/the-component-lifecycle/#toc_integrating-with-third-party-libraries-with-didinsertelement) hook.

File: app/pods/components/notification-dropdown/component.js [hunk 2]
```diff
@@ -24,13 +25,22 @@ export default class NotificationDropdownComponent extends Component {
 
   constructor () {
     super (...arguments)
+    const _this = this;
```
Comment (abhishek97) [inline app/pods/components/notification-dropdown/component.js]: could've used arrow functions instead

**Comment by abhishek97** [comment]
@Rishabh570 Please make the required changes and we'll merge this.

**Comment by Rishabh570** [comment]
@abhishek97 I somehow messed up here... Please take a look at my new PR regarding this (it contains latest changes).
I'm closing this one...

**Pull request closed without merging**